templates/cms_homepage.html

```diff
@@ -6,360 +6,89 @@
-  </figure>
+  </section>
 
-  <span class="description">The world's leading open source data management system. CKAN makes it easy to publish, share and discover data. It powers hundreds of data portals worldwide including governments, NGOs, research centers and enterprises.</span>
-  <em class="link-arrow-curved">
-    Learn more
-  </em>
-</a>
-<!-- /feature-item -->
+  <!-- section-homepage content  -->
+  {% placeholder "Homepage content" %}
 
-
-<!-- feature-item -->
-<a href="https://frictionlessdata.io/" target="_blank" class="feature-item ">
-  <figure class="">
-    <img src="/assets/images/home-tools-frictionless-data.png" alt="">
-  </figure>
-
-  <span class="description">For anyone who works with complex data or across tools or teams (researchers, data scientists, engineers, and decision-makers), this open-source toolkit brings simplicity to the data experience through software and standards.</span>
-  <em class="link-arrow-curved">
-    Learn more
-  </em>
-</a>
-<!-- /feature-item -->
-
-<!-- feature-item -->
-<a href="https://opendatacommons.org/"  target="_blank" class="feature-item">
-  <figure class="">
-    <img src="/assets/images/home-tools-open-data-commons.png" alt="">
-  </figure>
-
-  <span class="description">The home of a set of legal tools and licenses to help you publish, provide and use open data: Open Database License (ODbL), Open Data Commons Attribution License, and Open Data Commons Public Domain Dedication and License (PDDL).</span>
-  <em class="link-arrow-curved">
-    Learn more
-  </em>
-</a>
-<!-- /feature-item -->
+  <div class="sect -tighten-2 mb-20">
+    <div class="sect__content">
+      <div class="grid grid-cols-1 gap-10">
+        {% placeholder "Content Headlines" %}
+      </div>    
     </div>
   </div>
-</section>
-
 
-<!-- /section data tools -->
+  {% placeholder "Number stats block" %}
 
-<!-- section global network -->
-<section class="sect sect-hero -tighten-1 mx-auto -has-bg-circle -bg-circle-okfn-purple -bg-circle-lg mb-20">
-  <div class="sect__content">
-    <div class="lg:flex lg:gap-x-8 lg:items-center md:flex-row-reverse md:!gap-x-28">
-      <div class="flex-shrink-0">
-        <div class="flex-shrink-0 mb-10 lg:mb-0">
-          <img src="/assets/images/home-hero-global-movement.svg" class="mx-auto" alt="Home Hero">
-        </div>
-      </div>
-      <div class="flex-grow">
-        <h2 class="mb-8 font-bold text-h1 leading-none">Global <br class="md-no-break"> movement in <br class="md-no-break"> action</h2>
-        <span class="block mb-16 text-2xl">Open Knowledge Foundation is part of a worldwide movement working together for a fair, free and open future.</span>
-        <a href="/network" class="btn">Join us!</a>
+  <!-- section practice -->
+  <section class="sect sect-highlights mb-24">
+    <div class="sect__content">
+      <h2 class="mb-20 text-h1 text-center font-bold leading-none">Open Knowledge in <br> practice</h2>
```

Comment:
I would assume that this would need some changes since it will "hardcode" a title in the template that Lucas would not be able to change with plugins.